src/eos-google-chrome

```diff
@@ -94,6 +140,12 @@ class GoogleChromeLauncher:
         except OSError as e:
             exit_with_error("Could not launch Chrome: {}".format(repr(e)))
 
+        self._wait_for_installation()
```

Comment:
@joaquimrocha That would not suffice: what we want is that Chrome is set as the default browser after installing it, but what you're proposing would defer that until the user tries to run Chrome for the first time after installation.

It needs to be done on a post-inst event, what I'm not sure is where that logic should live. Any particular reason why we can't have a GS plugin that deals with that for Chrome?